### ISSUE TITLE
Wifi driver crashes on eduVPN disconnect

### ISSUE THREAD

**Issue by Jowsen** [Amebis/eduVPN#163]
Hi,

I recently got myself a wifi-card for my windows pc, which previously only had a LAN port. All good and well, however, sometimes when I disconnect the VPN (via the eduVPN desktop client, I wait for it to finish disconnecting before closing the app), the wifi driver suddenly fails and it may take several full system restarts to get it working again. Some information that may be useful to you, as I have no clue what would be required to pinpoint the problem, otherwise just contact me:
Device: Intel Wi-Fi 6E AX210 160MHz
Driver: Intel version 22.10.0.7 (up to date)
eduVPN version: 1.0.38.0

**Comment by fkooman** [comment]
Can you confirm that OpenVPN Communtiy from https://openvpn.net/community-downloads/ (2.5.2) works properly when connecting/disconnecting the VPN and doesn't crash your WiFi?

You can download an eduVPN configuration file for importing in OpenVPN Community through the eduVPN portal. This URL of the portal depends on your institution and can be found be locating your organization (or country) here: https://status.eduvpn.org/.

**Comment by Jowsen** [comment]
I'll try that one. It might take a while to reproduce as it seems to happen randomly, but did occur at least 4 times in the last two weeks.

**Comment by rozmansi** [comment]
For the record:

- eduVPN 1.0.38.0 is using
   - OpenVPN 2.5.1.12 aka [2.5.1-20210225](https://github.com/Amebis/openvpn/releases/tag/2.5.1-20210225) (our build)
   - TAP-Windows6 9.24.6

Another thing you can try is running eduVPN v2 client (in the _release candidate_ stage at the time of writing this). It uses more recent version of OpenVPN and the Wintun driver. With the different stack the problem will hopefully disappear.

**Comment by Jowsen** [comment]
Hi there,

Sorry for my late response, it took a while to reproduce the issue.

I got the issue once more using OpenVPN 2.5.2, after disconnecting the vpn, but it was resolve with a reboot. Today the device crashed on its own (no vpn connection active / no disconnecting of a connection), with the generic "device not found" (error code 10) message. I had to uninstall the drivers, reboot, reinstall, reboot, and then got it working again.

I think it's save to say the device drivers aren't stable and whatever is causing the crash may be triggered by disconnecting a vpn, but is not related to eduVPN.

Thanks for your responses and suggestions! It seems to occur less using the newest version of openVPN and I've kept the device drivers up to date as well, hoping for a patch.
